engine/src/main/java/org/destinationsol/game/SolGame.java

```diff
@@ -250,7 +245,8 @@ private void createMercs() {
         }
 
         Gson gson = new Gson();
-        Type type = new TypeToken<ArrayList<HashMap<String, String>>>() {}.getType();
+        Type type = new TypeToken<ArrayList<HashMap<String, String>>>() {
```

Comment:
Please no. The point of formatting is to make code easier to read, not split it across multiple line unnecessarily.

Reply:
Something got accidentallied there

Reply:
Done.